<?xml version="1.0"?>
<launch>
    <include file="$(find realsense2_camera)/launch/rs_camera.launch">
        <arg name="camera"          value="external_camera"/>
        <arg name="serial_no"       value="740112070308"/>
        <arg name="depth_width"     value="1280"/>
        <arg name="depth_height"    value="720"/>
        <arg name="enable_depth"    value="false"/>
        <arg name="color_width"     value="1280"/>
        <arg name="color_height"    value="720"/>
        <arg name="enable_color"    value="true"/>
    </include>

    <include file="$(find realsense2_camera)/launch/rs_rgbd.launch">
        <arg name="camera"          value="hand_camera"/>
        <arg name="serial_no"       value="838212074282"/>
        <arg name="depth_width"     value="1280"/>
        <arg name="depth_height"    value="720"/>
        <arg name="color_width"     value="1280"/>
        <arg name="color_height"    value="720"/>
    </include>
</launch>